service/RTC/SignallingLayer.js

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
nit: in the states we'd spell this "SignalingLayer" (one "L" in signaling).  apparently "signalling" is the UK way which probably explain why makes my brain feel weird when i read it.